src/platform/elements/form/NovoFormControl.ts

```diff
@@ -195,7 +206,9 @@ export class NovoFormControl extends FormControl {
     }
 }
 
+
 export class NovoFormGroup extends FormGroup {
+    @Output() public novoConfigChange: EventEmitter<any> = new EventEmitter();
```

Comment:
No output here, and make a specific interface for the data model. Make sure to export the interface inside of `index.ts`